### PR TITLE
cloud-controller-manager and kubelet glossary file updated with corresponding links

### PR DESCRIPTION
**Pull request by tamilselvan1102**
cloud-controller-manager and kubelet glossary files has been updated with corresponding links
PR #40820

### CODE REVIEW

File: content/en/docs/reference/glossary/cloud-controller-manager.md [hunk 1]
```diff
@@ -19,6 +19,6 @@ with that cloud platform from components that only interact with your cluster.
 <!--more-->
 
 By decoupling the interoperability logic between Kubernetes and the underlying cloud
-infrastructure, the cloud-controller-manager component enables cloud providers to release
+infrastructure, the [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/) component enables cloud providers to release
```
Comment (mengjiao-liu) [inline content/en/docs/reference/glossary/cloud-controller-manager.md:22]: Why not update the link at line 15?

It's more noticeable in the front, isn't it?

**Comment by mengjiao-liu** [comment]
Preview pages:
Cloud Controller Manager: https://deploy-preview-40822--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?all=true#term-cloud-controller-manager

Kubelet: https://deploy-preview-40822--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?all=true#term-kubelet

File: content/en/docs/reference/glossary/cloud-controller-manager.md (hunk 1 above)
Comment (tamilselvan1102) [reply content/en/docs/reference/glossary/cloud-controller-manager.md:22]: Till line 17 is using for glossary tooltip messages and hence created a link in the _more_ (line 19) section.

Comment (dipesh-rawat) [reply content/en/docs/reference/glossary/cloud-controller-manager.md:22]: @tamilselvan1102 The tooltip on Line 14 pertains only to the term 'control plane'. As previously suggested in a review comment, Line 15 may accommodate the inclusion of a link for 'cloud-controller-manager'.

Comment (tamilselvan1102) [reply content/en/docs/reference/glossary/cloud-controller-manager.md:22]: @dipesh-rawat 
 I have investigated,  [cloud-controller-manager] is not used anywhere as tooltip but used as [glossary_definition term_id] in the following pages. So accepted the suggestion and will update the PR.

https://kubernetes.io/docs/concepts/architecture/cloud-controller/
https://kubernetes.io/docs/concepts/overview/components/
https://kubernetes.io/docs/tasks/administer-cluster/controller-manager-leader-migration/
https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/

**Comment by dipesh-rawat** [comment]
@tamilselvan1102  I see the Netlify build failed. Could you try to trigger the build again using:
`git commit --amend --no-edit && git push --force-with-lease`

**Comment by tamilselvan1102** [comment]
> @tamilselvan1102 I see the Netlify build failed. Could you try to trigger the build again using: `git commit --amend --no-edit && git push --force-with-lease`

**Comment by tamilselvan1102** [comment]
Netlify build keep on failing so closed this and created new PR . Sorry for the inconvenient
#40931

**Pull request closed without merging**